schemas/input/csl-data.json

```
@@ -4,425 +4,477 @@
+          "type": "string"
+        },
+        "event-place": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "first-reference-note-number": {
+          "type": ["string", "number"]
+        },
+        "genre": {
+          "type": "string"
+        },
+        "ISAN": {
+          "type": "string"
+        },
+        "ISBN": {
+          "type": "string"
+        },
+        "ISCI": {
+          "type": "string"
+        },
+        "ISMN": {
+          "type": "string"
+        },
+        "ISRC": {
+          "type": "string"
+        },
+        "ISSN": {
+          "type": "string"
+        },
+        "ISWC": {
+          "type": "string"
+        },
+        "issue": {
+          "type": ["string", "number"]
+        },
+        "jurisdiction": {
+          "type": "string"
+        },
+        "keyword": {
+          "type": "string"
+        },
+        "locator": {
+          "type": ["string", "number"]
+        },
+        "medium": {
+          "type": "string"
+        },
+        "note": {
+          "type": "string"
+        },
+        "number": {
+          "type": ["string", "number"]
+        },
+        "number-of-pages": {
+          "type": ["string", "number"]
+        },
+        "number-of-volumes": {
+          "type": ["string", "number"]
+        },
+        "original": {
+          "title": "Original - Related Item",
+          "description": "Used to define metadata for an original publication.",
+          "$ref": "#/definitions/bibitem"
+        },
+        "original-publisher": {
+          "description": "[Deprecated - Use `publisher` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-publisher-place": {
+          "description": "[Deprecated - Use `publisher-place` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-title": {
+          "description": "[Deprecated - Use `title` on the `original` related item instead]",
+          "type": "string"
+        },
+        "page": {
+          "type": ["string", "number"]
+        },
+        "page-first": {
+          "type": ["string", "number"]
+        },
+        "part": {
+          "type": ["string", "number"]
+        },
+        "part-title": {
+          "type": "string"
+        },
+        "PMCID": {
+          "type": "string"
+        },
+        "PMID": {
+          "type": "string"
+        },
+        "printing": {
+          "type": ["string", "number"]
+        },
+        "publisher": {
+          "type": "string"
+        },
+        "publisher-place": {
+          "type": "string"
+        },
+        "references": {
+          "type": "string"
+        },
+        "reviewed": {
+          "title": "Reviewed - Related Item",
+          "description": "In a review, defines metadata for the reviewed item.",
+          "$ref": "#/definitions/bibitem"
```

```suggestion
          "$ref": "#/definitions/related-bibitem"
```

This and `original` shoudl be `related-bibitem`, no?